daemon/pod.go

```diff
@@ -422,7 +418,8 @@ func (p *Pod) createVolumes(daemon *Daemon) error {
 
 	sd := daemon.Storage
 	for i := range p.spec.Volumes {
-		if p.spec.Volumes[i].Source == "" {
+		// hyperd doesn't 'remove' vfs volume
+		if p.spec.Volumes[i].Driver != "vfs" {
 			vol, err = sd.CreateVolume(daemon, p.id, p.spec.Volumes[i].Name)
```

Comment:
I think this patch changed the semantics of the `CreateVolume`. In the previous code, the daemon check the user spec, and only create volume when the user doesn't specify the source of the volume.